docker/tensorflow-aarch64/Dockerfile

```diff
@@ -268,22 +249,18 @@ ENV VENV_DIR=/home/$DOCKER_USER/python3-venv
 COPY --chown=$DOCKER_USER:$DOCKER_USER --from=tensorflow-tools $VENV_DIR /home/$DOCKER_USER/python3-venv
 ENV PATH="$VENV_DIR/bin:$PATH"
 
-# Build Bazel
-COPY scripts/build-bazel.sh $PACKAGE_DIR/.
-RUN $PACKAGE_DIR/build-bazel.sh
-ENV PATH=$PACKAGE_DIR/bazel/output:$PATH
+# Get Bazel binary for AArch64
+COPY scripts/get-bazel.sh $PACKAGE_DIR/.
+RUN $PACKAGE_DIR/get-bazel.sh
```

Comment:
This file get-bazel.sh seems to be missing from this repository.

Reply:
Yep, well spotted. There was slight mix-up. It is there but erroneously named `get-onednn.sh`.

There's a fix up now: https://github.com/ARM-software/Tool-Solutions/commit/1b0e58ba93ca19fcf8f48af5c135111c5cc897a7